data/News/2021-06-22-tickets-on-sale.md

```diff
@@ -0,0 +1,24 @@
+---
+title: CFP closed, Tickets on sale
+published: 2021-06-22
+---
+
+We have been overwhelmed by responses to our call for proposals for the second very online PyConline. Our program team is currently reviewing MANY submissions, and curating a schedule for our Friday specialist tracks and Saturday general conference. We anticipate everyone who submitted will hear from us about their talk within one month of the CFP close, or **before July 10 2021 **(hopefully earlier!)
+
+While you wait, we want to get you excited about PyConline AU regardless of the program. With that, we're happy to announce **[tickets are now available](2021.pycon.org.au/attend)**!** **Our prices remain consistent with last year, with Enthusiast tickets at $29,  Professionals at $79, and Contributors at $300. Students can attend for only $10 upon confirmation of a valid student ID. All prices in AUD including GST.
```

Comment:
The link here isn't complete - I've shortened it to a relative link, if that works?
```suggestion
While you wait, we want to get you excited about PyConline AU regardless of the program. With that, we're happy to announce **[tickets are now available](/attend)**!** **Our prices remain consistent with last year, with Enthusiast tickets at $29,  Professionals at $79, and Contributors at $300. Students can attend for only $10 upon confirmation of a valid student ID. All prices in AUD including GST.
```